### PR TITLE
Add warning to SolarEdge local documentation

### PR DESCRIPTION
**Pull request by drobtravels**
This integration no longer functions with recent firmware updates.  It may continue work long term on inverters which do not receive firmware updates

More details in [this issue](https://github.com/drobtravels/solaredge-local/issues/24)

## Proposed change

Add warning and reference to issue with more details so users can quickly determine if this integration will work for them.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ X]This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/solaredge_local.markdown [hunk 1]
```diff
@@ -12,17 +12,21 @@ ha_codeowners:
 ha_domain: solaredge_local
 ---
 
+<div class='note'>
+
+Recent firmeware updates have disabled the local API on many inverters.  Please enter the IP address of your inverter in a browser before attempting to use this component.  If the local API is enabled, you'll see a HTML page with the SolarEdge logo and a "Commissioning" menu.  See [this issue](https://github.com/jbuehl/solaredge/issues/124) and [this issue](https://github.com/drobtravels/solaredge-local/issues/24) for additional details.
```
Comment (klaasnicolaas) [inline source/_integrations/solaredge_local.markdown]:
```suggestion
Recent firmeware updates have disabled the local API on many inverters. Please enter the IP address of your inverter in a browser before attempting to use this component. If the local API is enabled, you'll see a HTML page with the SolarEdge logo and a "Commissioning" menu. See [this issue](https://github.com/jbuehl/solaredge/issues/124) and [this issue](https://github.com/drobtravels/solaredge-local/issues/24) for additional details.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
Should this be an alert on alerts.home-assistant.io?

**Comment by drobtravels** [comment]
Thanks for reviewing.  I'm unclear if any action is required by me on this?  I see changes have been requested, but it looks like they were already applied.

I can add an alert to the PR if you would like.  I was unfamiliar with that site.  Note that this has been an issue for a long time, but I just got around to adding an update.

**Review by frenck — APPROVED** [review]
Thanks, @drobtravels!

I've adjusted a bit and moved the warning a bit down (let's start with an introduction before serving the user a warning 😉 )

**Pull request merged**